src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    Lottery getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor().name(), new Random().nextInt(100));
+    }
```

Comment:
Better make Random and ColorSupplier class fields. The best way to initialize them - is using constructor. Initialization when you declare is also possible for this task.
```suggestion
    private Random random;
    private ColorSupplier colorSupplier;
    
    Lottery getRandomBall() {
        return new Ball(colorSupplier.getRandomColor().name(), random.nextInt(100));
    }
```
**And 100 is a magic number - make it constant!**